recipe-client-addon/lib/NormandyDriver.jsm

```diff
@@ -156,6 +156,68 @@ this.NormandyDriver = function(sandboxManager) {
       sandboxManager.removeHold(`setTimeout-${token}`);
     },
 
+    /*
+     * Return a promise that resolves to an Addon ID if installation is successful.
+     */
+    installAddon(url) {
+        /* Install an addon.  Note that the AddonManager can only handle
+         * file:// and downloadable URLs.
+         */
+
+      var installObjectCallback = (installObject, resolve, reject) => {
```

Comment:
`let`, here and elsewhere. Though this also doesn't actually seems to use its binding (ie `this`) and so maybe you just want:

```js
function installObjectCallback(installObject, resolve, reject) {
...
}
```